packages/mui-lab/src/Masonry/Masonry.test.js

```diff
@@ -112,9 +113,14 @@ describe('<Masonry />', () => {
       if (/jsdom/.test(window.navigator.userAgent)) {
         this.skip();
       }
-      expect(() => render(<Masonry columns={3} spacing={1} />)).toErrorDev(
-        'Warning: Failed prop type: The prop `children` is marked as required in `ForwardRef(Masonry)`, but its value is `undefined`.',
-      );
+
+      // React 19 removed prop types support
+      if (reactMajor < 18) {
```

Comment:
```suggestion
      if (reactMajor < 19) {
```

Or am I missing something?

Reply:
🤦 you're correct